sys/include/xtimer.h

```diff
@@ -47,7 +47,7 @@ extern "C" {
  * @note This is a struct in order to make the xtimer API type strict
  */
 typedef struct {
-    uint64_t ticks64;
+    uint64_t ticks64;       /**< Captain Obvious to the rescue! */
```

Comment:
I have different opinion, but it's more important to get this merged ASAP than having a long discussion about opinions, so I removed the fun.